proxy-bosh-release/jobs/wavefront_proxy/templates/wavefront.conf.erb

```diff
@@ -30,6 +31,11 @@ persistMessages=false
 preprocessorConfigFile=/var/vcap/jobs/wavefront_proxy/conf/preprocessor_rules.yaml
 <% end %>
 
+<% if logsIngestion && (logsIngestion.length > 0) %>
+rawLogsPort=5045
```

Comment:
It'd be preferable to not hard code this port. Users can add this property manually in the custom configuration.

Suffice to simply allow them to enter the logsIngestion.yaml contents as you have done.